tests/testapp/tests/test_filter_classes.py

```diff
@@ -244,3 +245,14 @@ def test_filter_by_org_managed_with_org_field(self):
             url, args=(lib1.id), HTTP_AUTHORIZATION=f'Bearer {token}'
         )
         self.assertEqual(response.status_code, 404)
+
+    def test_book_nested_shelf(self):
+        operator = self._get_operator()
+        self._create_org_user(
+            user=operator, is_admin=True, organization=self._get_org('org_a')
+        )
+        token = self._obtain_auth_token(operator)
+        with self.assertNumQueries(0):
+            url = reverse('test_book_nested_shelf')
+        response = self.client.get(url, HTTP_AUTHORIZATION=f'Bearer {token}')
```

Comment:
Wait, `self.client.get(url, HTTP_AUTHORIZATION=f'Bearer {token}')` should be inside `self.assertNumQueries(0)`, can you please explain this?

Reply:
Ajay is right ,there's a mistake here

Reply:
Ohh sorry, will try to rectify it ASAP.